freqtrade/exchange/exchange.py

```diff
@@ -1318,8 +1288,8 @@ def refresh_latest_ohlcv(self, pair_list: ListPairsWithTimeframes, *,
                 self._pairs_last_refresh_time[(pair, timeframe)] = ticks[-1][0] // 1000
             # keeping parsed dataframe in cache
             ohlcv_df = ohlcv_to_dataframe(
-                    ticks, timeframe, pair=pair, fill_missing=True,
-                    drop_incomplete=self._ohlcv_partial_candle)
+                ticks, timeframe, pair=pair, fill_missing=True,
+                drop_incomplete=self._ohlcv_partial_candle)
```

Comment:
The autopep8 PR will take care of all the annoying changes like this, so that they don't clutter up future PRs